recipes/triqs_tprf/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "2.2.0" %}
+
+package:
+  name: triqs_tprf
+  version: {{ version }}
+
+source:
+  url: https://github.com/TRIQS/tprf/releases/download/{{ version }}/tprf-{{ version }}.tar.gz
+  sha256: 87389c71bdf690e05b72a5cd7dd2f4f7d8a8ac69b15f1481de1960e7740efc9f
+
+build:
+  number: 0
+  skip: True  # [win or py>30]
+  merge_build_host: True
+
+requirements:
+  build:
+    - cmake
+    - make
+    - clangxx >=8  # [osx]
```

Comment:
You shouldn't need to install compilers directly

Reply:
We require a recent version of clang however, since we are making use of the C++17 standard.
Will it be guaranteed that a recent clang version is used for compilation even without this?

Reply:
You can set the standard used with `export CXXFLAGS="${CXXFLAGS} -std=c++17"` or similar. From [here](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L24) you can see that we use clang v9

Reply:
Flags should be forwarded from the triqs library cmake target that we link against. I removed this and it seems to pass just fine.